filer/migrations/0005_default_alt_text_size.py

```diff
@@ -0,0 +1,15 @@
+from django.db import migrations, models
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('filer', '0004_unique_clipboard_per_user'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='Image',
+            name='default_alt_text',
```

Comment:
this is ok, it will fix the Jira issue, but I think we should do the same for the `default_caption`. I imagine that if someone tried to put a long string for the alt text, it will do the same for the image caption.

Reply:
"plugin to add context; there is a 140-character limit," that is what is specified in the imagemodels.py. If this is right we can go as it is

Reply:
I think that's outdated. Even if there is such a plugin, that's fine if it will show only 140 chars. On the other hand, the image `default_caption` is used in our components like the image component: https://bento3-staging.pbs.org/bento-cms/preview/673466/. The caption div has an automatic size depending on the caption length.

Reply:
btw, there's a Validation for the default_caption max chars to 140, but that validation is totally skipped right now when updating the Image through django-filer-rest.  (https://github.com/pbs/django-filer/blob/master_pbs/filer/models/imagemodels.py#L98)

Reply:
in fact non of the validation seems to work, fixed for this one we probably need a new ticket for the others